release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -3351,3 +3351,19 @@ There are no images for {product-title} 4.7.53.
 ==== Updating
 
 To update an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-7-54"]
+=== RHSA-2022:4965 - {product-title} 4.7.54 bug fix and security update
```

Comment:
Looks like this is an RHBA rather than a security update. If that is the case it should  be:
```suggestion
=== RHBA-2022:5505 - {product-title} 4.7.54 bug fix and security update
```

Reply:
updated!